client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureSearchTextField/ExposureSearchTextFieldStyles.ts

```diff
@@ -0,0 +1,11 @@
+import {makeStyles} from '@material-ui/styles';
+
+import theme from 'styles/theme';
+
+const useStyles = makeStyles({
+    serachIcon: {
+        color: '#107F9B'
```

Comment:
Where did you get this color from?

Reply:
I can see it's pretty similar to the theme's primary color. You can try using the color='primary' prop instead on the IconButton.

Reply:
from here https://dev.azure.com/spectrumFactory/CoronaI/_sprints/backlog/CoronaI%20Team/CoronaI/Sprint%2013?workitem=1247
> בלחיצה על אנטר או כפתור החיפוש(search מטיריאל UI בצבע כחול (#107F9B)) יופיע תוצאות מתאימות שבועיים אחורה בלבד מיום תאריך המחלה של הנחקר!

it does says material-ui blue but there's no shade of blue in material ui that is #107F9B 🤔